RF24.h

```diff
@@ -1277,11 +1277,11 @@ class RF24
      * @code
      * bool goodSignal = radio.testRPD();
      * if(radio.available()){
-     *    Serial.println(goodSignal ? "Strong signal > 64dBm" : "Weak signal < 64dBm" );
+     *    Serial.println(goodSignal ? "Strong signal > -64dBm" : "Weak or no signal < -64dBm" );
```

Comment:
```suggestion
     *    Serial.println(goodSignal ? "Strong signal > -64dBm" : "Weak or no signal (< -64dBm)" );
```
Just so it doesn't read like "no signal was less than -64 dBm"